lang/lang/syntax/constructors.sdf3

```diff
@@ -0,0 +1,168 @@
+module constructors
+
+imports
+
+  lex/literal
+  pie
+//  statix/API // use for Stx-Scope
```

Comment:
There's a lot of commented out/TODO in this file. Do you intend to resolve that in a future PR? If not, please resolve them now.

Reply:
Should it be resolved: yes.
Will it be resolved by me? Likely not.
The problem is that I couldn't get it to work, and I cannot afford to waste more time on it.

Reply:
I have removed them (/moved documentation to Statix constructors) and created an issue: meamanusername/pie#102